crypto/bls_crossBLST_test.go

```diff
@@ -0,0 +1,205 @@
+// +build relic
+
+package crypto
+
+// This file contains tests against the library BLST (https://github.com/supranational/blst).
```

Comment:
Putting the file move, on the one hand, and the comments + modifications, on the other hand, in distinct commits would help tremendously towards reviewing this file.

Reply:
You're right, my bad 🤦  
The tests scenarios were left as they were, there was mainly lots of renaming (mainly use `flow` instead of `bls` to refer to the local library)